cli/cli/helpers/portal_manager/binary_artifact_getter.go

```diff
@@ -54,48 +56,60 @@ func DownloadLatestKurtosisPortalBinary(ctx context.Context) (bool, error) {
 	}
 
 	ghClient := github.NewClient(http.DefaultClient)
-	latestRelease, err := getLatestRelease(ctx, ghClient)
+	var release *github.RepositoryRelease
+	if portalTag == portalTagLatest {
```

Comment:
Can't we just call `getReleaseByTag` and if the tag is `latest` we just pull latest? Seems overcomplicated to have 2 different functions here. Maybe there's some difficulty I'm missing